packages/console/src/App.tsx

```diff
@@ -73,7 +75,10 @@ const Main = () => {
             <Route path="callback" element={<Callback />} />
             <Route path="welcome" element={<Welcome />} />
             <Route element={<AppLayout />}>
-              <Route path="/cloud-preview" element={<CloudPreview />} />
+              <Route path="cloud-preview" element={<CloudPreview />}>
```

Comment:
questionnaire?

Reply:
The questionnaire is part of the preview content and is under the cloud preview page.